src/resolvers/default.rs

```diff
@@ -307,6 +326,71 @@ impl Hash for HashBLAKE2s {
     }
 }
 
+#[cfg(feature = "pqclean_kyber1024")]
+impl Default for Kyber1024 {
+    fn default() -> Self {
+        Kyber1024 {
+            pubkey:  kyber1024::PublicKey::from_bytes(&[0; 1568]).unwrap(),
+            privkey: kyber1024::SecretKey::from_bytes(&[0; 3168]).unwrap(),
```

Comment:
what are these magic numbers from? do they exist as constants elsewhere?

Reply:
They do, for example https://docs.rs/pqcrypto/0.6.0/pqcrypto/kem/kyber768/fn.secret_key_bytes.html

Reply:
Yes, the PQClean crate seems to have these values in `const` functions. Fixed.